### ISSUE TITLE
Specter Origin Can't Go Through Portals

### ISSUE THREAD

**Issue by Nexus71c** [The-Architects727/DnDMonsterOrigins#1]
I'm hosting a server where we're using these Origins, and a friend of mine is using the Specter origin.
Everything works as intended, but for some reason they're unable to enter portals of any kind. They've tried the Nether, End, and also Deeper & Darker portal (as we have that installed), none of them actually work. They can sit in each portal and never be teleported.
So far, I've had to use command blocks and teleport them which doesn't always work and teleports others..

TLDR: Specter origin can't travel to other dimensions via portals. Any fix would be appreciated!

**Comment by The-Architects727** [comment]
Sorry about that. When it says that you can phase through all blocks, I forgot to blacklist the portal blocks. I will get on a fix as soon as I can, though that will be a bit out.
